plugin.rb

```diff
@@ -6,6 +6,8 @@
 # author: Vinoth Kannan
 # url: https://github.com/discourse/discourse-salesforce
 
+gem 'jwt', '2.3.0'
```

Comment:
Core already has this gem. By referencing it here, it means we will have errors when we bump the version in core. Could we just remove it, and rely on core having it installed already? (we use JWT in a number of plugins, including the very popular openid-connect plugin)